main.js

```diff
@@ -1,32 +1,41 @@
+let url = window.location.toString();
 
-
-let str = "https://github.com/?username=6thSence";
 let nickName = (link) => {
   let arr = link.split('=');
-  return (arr[arr.length-1]);
+  let nick = arr[1];
+  if(nick === undefined) {
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/WebHeroSchool/jstask35/pullRequest?prid=6095927)